src/data/events/archive/winter2022/hoth-9.yml

```
@@ -0,0 +1,91 @@
+- name: Hack off the Hill 9 Workshops
+  quarter: Winter 2022
+  mainLink: https://www.youtube.com/watch?v=GskV5qMOpkE&list=PLPO7_kXilXFbjHembKB9UdcMyFkc_1Q06
+  tags: ["hackathon"]
+  directors:
+    - Christina Tong
+    - Einar Balan
+
+  workshops:
+    - name: Intro to Hackathons
+      youtube: https://www.youtube.com/watch?v=GskV5qMOpkE
+      tags: ["hackathon"]
+      presenters:
+        - Rajeshwari Jadhav
+    - name: Intro to HTML
+      youtube: https://youtu.be/qQy8lwOHkBY
+      repo: https://github.com/uclaacm/hoth8-intro-to-html-css-js
+      slides: https://docs.google.com/presentation/d/1Y9rvAXjyeZOADBceqlAr9eeakO7URGWwu4QRulgvhSs/edit
+      tags: ["html"]
+      presenters:
+        - Jakob Reinwald
+    - name: Intro to CSS
+      youtube: https://youtu.be/DdY_VfwJZ1w
+      repo: https://github.com/uclaacm/hoth8-intro-to-html-css-js
+      slides: https://docs.google.com/presentation/d/1Y9rvAXjyeZOADBceqlAr9eeakO7URGWwu4QRulgvhSs/edit
+      tags: ["css"]
+      presenters:
+        - Jakob Reinwald
+    - name: Intro to JavaScript
+      youtube: https://youtu.be/Q23CBrln3SA
+      repo: https://github.com/uclaacm/hoth8-intro-to-html-css-js
+      slides: https://docs.google.com/presentation/d/1Y9rvAXjyeZOADBceqlAr9eeakO7URGWwu4QRulgvhSs/edit
+      tags: ["javascript"]
+      presenters:
+        - Jakob Reinwald
+    - name: "Intro to Web Development: React.js"
+      youtube: https://www.youtube.com/watch?v=4YZ0KZfCcb8&list=PLPO7_kXilXFbjHembKB9UdcMyFkc_1Q06&index=7
```

```suggestion
      youtube: https://www.youtube.com/watch?v=4YZ0KZfCcb8
```

Remove end of url so it doesn't load the playlist as well